pkgs/tools/audio/beets/default.nix

```diff
@@ -215,13 +232,7 @@ in pythonPackages.buildPythonApplication rec {
       s,"mp3val","${mp3val}/bin/mp3val",
```

Comment:
And the changes here and above as well, part of a patch and not a sed command.. But this should not block the PR if you don't want to do it @lovesegfault .

Reply:
Same as the above.